lib/workers/global/config/parse/file.ts

```diff
@@ -1,8 +1,28 @@
+import { readFile } from 'fs-extra';
+import { load } from 'js-yaml';
+import JSON5 from 'json5';
 import upath from 'upath';
 import { migrateConfig } from '../../../../config/migration';
-import type { AllConfig } from '../../../../config/types';
+import type { AllConfig, RenovateConfig } from '../../../../config/types';
 import { logger } from '../../../../logger';
 
+export async function getParsedContent(file: string): Promise<RenovateConfig> {
+  switch (upath.extname(file)) {
+    case '.yaml':
+    case '.yml':
+      return load(await readFile(file, 'utf8'), {
+        json: true,
+      }) as RenovateConfig;
+    case '.json5':
```

Comment:
```suggestion
    case '.json5':
    case '.json':
```

I would prefer to use this for all json files